tracer/src/Datadog.Trace/PlatformHelpers/ContainerMetadata.cs

```diff
@@ -23,7 +21,7 @@ namespace Datadog.Trace.PlatformHelpers
     /// <summary>
     /// Utility class with methods to interact with container hosts.
     /// </summary>
-    internal static class ContainerMetadata
+    internal class ContainerMetadata
```

Comment:
```suggestion
    internal sealed class ContainerMetadata
```

Reply:
Yeah but then how do you mock the value of the properties in tests ? My idea was that you especially wouldn't want this sealed, so that tests can create mock of this.
Or I can create an interface `ContainerMetadata` that this implements.